### ISSUE TITLE
Is Gender and Name considered to be part of an NPC's "Appearance" ?

### ISSUE THREAD

**Issue by sbstnkhlr** [focustense/easymod#134]
I was playing around a bit with EasyNPC to see how it handles some uncommon cases of appearance replacement. Most overhauls keep Name and Gender consistent with the Default Plugin, so I tried to "break" the system by making a replacer that changes an NPC's gender. 

Unless I did something wrong, it seems EasyNPC sticks to the gender of the Default Plugin and doesn't take a possible gender swap from the NPC replacer into consideration. As a result I got a darkfaced npc with the replacement gender face and default gender body.

It might be worth it to have EasyNPC consider such cases. I don't know of any specific NPC replacer out there that does this, but it's probably not unlikely that they exist.

TL;DR- Consider having EasyNPC take gender and also possibly the name from the replacer instead of the default plugin.
